src/content/docs/infrastructure/host-integrations/host-integrations-list/nuxt-js-integration.mdx

```diff
@@ -0,0 +1,83 @@
+
+With our Nuxt.js dashboard, you can easily track your application error rate, initial page load average, route changes, throughput, and page views with JavaScript errors. Built with our infrastructure agent and our Vue browser monitoring, the Nuxt.js dashboard lets you view critical information all in one place.
+
+<img
+  title="Nuxt.js dashboard"
+  alt="A screenshot of a Nuxt.js dashboard"
+  src={nuxtjsdashboard}
+/>
+
+## 1. Install infrastructure agent [#infra]
+Before you can get your data into New Relic, you should first install the infrastructure agent (if you have it installed, you can skip this step). This enables New Relic to collect data so that application error rates, throughput, and page views with JavaScript errors can appear in dashboards. This step is optional, but we strongly recommend you install the infrastructure agent to see all your host and browser monitoring data.
+
+There are two ways to install the infrastructure agent:
+* Follow the [guided install](https://one.newrelic.com/marketplace?state=607786be-99a9-fa66-4b67-3f7f3b9551d4) to instrument our system with the infrastructure agent.
+* [Install the infrastructure agent](docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) using the command line. 
+
+
+## 2. Install Vue browser monitoring [#install]
+After installing the infrastructure agent, the next step is to install the browser monitor. From **[one.newrelic.com](https://one.newrelic.com)**, click **Add data > Browser & mobile > Vue**.
+Follow the instructions from there to generate the JavaScript code snippet used in the next step.
+
+## 3. Configure the integration [#config]
+To add the JavaScript snippet in the Nuxt.js application:
+1. Create a JavaScript file in the `static` folder:
+  ```sql
+  touch <FILENAME>.js
+  ```
+2. Create a function in the newly created JavaScript file to execute the generated script:
+  ```sql
+  nano <FILENAME>.js
+  ```
+  ```sql
+  function <FUNCTION_NAME>() {
+	<GENERATED SCRIPT IS PASTED HERE BY REMOVING OPENING AND CLOSING <script> TAGS>
+  }
+  FUNCTION_NAME();
+  ```
+3. Add the created JavaScript to the nuxt configuration file named `nuxt.config.js`:
+  ```sql
+  nano nuxt.config.js
+  ```
+4. Add the following snippet into the `nuxt.config.js` file:
+  ```sql
+     __dangerouslyDisableSanitizers: ["script"],
+   script: [
+     {
+       hid: "<FUNCTION_NAME>",
+       src: "/<FILENAME>.js",
+       defer: true,
+       type: "text/javascript",
+       charset: "utf-8",
+     },
+   ],
+  ```
+5. Restart your application.
+
+## 4. Monitor your application on one.newrelic.com [#monitor]
+
+Now, you should be able to see your application on **[one.newrelic.com](https://one.newrelic.com)** by selecting **Browser** from the navigation bar. Choose your application to see all the metrics which are captured on the **one.newrelic.com** platform. You can also view the data with NRQL using the following query:
```

Comment:
View Nuxt.js metrics on newrelic's pre built dashboard:
1. Go to Add Data from one.newrelic.com
2. Search for nuxtjs in Dashboards section.
3. Install the nuxtjs dashboard.
4. Then you can see your dashboard in [one.newrelic.com](https://one.newrelic.com/all-capabilities) -> ***Dashboards*** section